website/src/content/docs/blog/biome-wins-prettier-challenge.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Biome formatter wins the Prettier challenge
+template: splash
+url: /blog/biome-wins-prettier-challenge
+description: Biome formatter now 95% compatible with Prettier
+summary: Biome formatter now 95% compatible with Prettier
+authorName: Emanuele Stoppa
+authorAvatar: https://avatars.githubusercontent.com/u/602478?v=4
+authorUrl: https://twitter.com/ematipico
+coauthorName: Biome Core Team, Biome Maintainers
+coauthorAvatar: /img/circle-indent-logo.svg
+pubDate: 2023-11-27
+layout: ../../../layouts/Blog.astro
+coverImage: blog/prettier-challenge.png
+coverImageAltText: The Prettier challenge banner, with the Biome logo over it
+socialImage: blog/prettier-challenge.png
+---
+
+With the release of Biome **`v1.4.0`**, we claim the bounty of the [Prettier challenge](https://console.algora.io/challenges/prettier)!
+
+With `v1.4.0`, you'll get a better formatter experience, more formatting options, new VSCode features, new sponsors and more!
+
+You can upgrade Biome by running the following command:
+
+```bash
+npm install --save-dev --save-exact @biomejs/biome@1.4.0
+pnpm update --save-exact @biomejs/biome@1.4.0
+yarn upgrade --exact @biomejs/biome@1.4.0
+```
+
+##  Better formatter
+
+Biome formatter has now **over 96% in terms of compatibility** against [Prettier](https://prettier.io/)!
```

Comment:
> > Any contribution is welcomed :)
>
> Where should we have this phrase? How should we phrase it?

Maybe at the start? We could just say 95% for these 3 languages. The displayed score on the challenge page corresponds to the global score and thus includes these 3 languages.

```suggestion
Biome formatter has now **over 95% in terms of compatibility** against [Prettier](https://prettier.io/)!
This score is computed for JavaScript, TypeScript, and JSX formatting.
```